doc/manuals-ld/troe.md

```diff
@@ -7,3 +7,50 @@ The sink used for TRoE is Postgres, with PostGIS and TimescaleDB extensions.
 While Orion-LD takes care of populating the TRoE databases, another component handles the queries of temporal data - [Mintaka](https://github.com/FIWARE/Mintaka).
 
 So, for queries of the temporal data, instead of sending the requests to Orion-LD, on (default) port 1026, the queries are sent to Mintaka, on (default) port xxxx.
+
+## Database setup
+
+For running orion-ld with TroE enabled, you need to setup a PostgreSQL with PostGIS and TimescaleDB. For local installations, we can recommend the 
+[timescaledb-postgis image](https://hub.docker.com/layers/timescale/timescaledb-postgis/latest-pg12/images/sha256-40be823de6035faa44d3e811f04f3f064868ee779ebb49b287e1c809ec786994?context=explore).
+
+You can start it with 
+```
+docker run -e POSTGRES_USER=orion -e POSTGRES_PASSWORD=orion -e POSTGRES_HOST_AUTH_METHOD=trust timescale/timescaledb-postgis:latest-pg12
+```
+
+## Database migration
+
+You can find the databse migration scripts ins the [database-folder](../../database)   
```

Comment:
Typo: databse.

Also, we don't want any 'we' and 'you' in the documentation, except for "special occasions".
 I'll go over it shortly.

Reply:
Fixed in 0b6b6e2